docker-org-and-tag/action.yml

```diff
@@ -29,49 +34,59 @@ runs:
+            elif [ "$GITHUB_BASE_REF" == "carma-master" ] || [ "$GITHUB_BASE_REF" == "master" ] || [[ -n "$GIT_TAG" ]]; then
                 echo "PR against master ..."
-                echo docker_organization="usdotfhwastol" >> $GITHUB_OUTPUT
-                echo docker_image_tag="$GIT_TAG" >> $GITHUB_OUTPUT
+                echo "docker_organization=usdotfhwastol" >> $GITHUB_OUTPUT
+                docker_image_tag="$GIT_TAG"
             else
-                echo docker_organization="usdotfhwastoldev" >> $GITHUB_OUTPUT
+                echo "docker_organization=usdotfhwastoldev" >> $GITHUB_OUTPUT
                 if [ "$GITHUB_BASE_REF" == "carma-develop" ] || [ "$GITHUB_BASE_REF" == "develop" ]; then
                     echo "PR against develop ..."
-                    echo docker_image_tag="develop" >> $GITHUB_OUTPUT
+                    docker_image_tag="develop"
                 else
                     echo "PR against $GITHUB_BASE_REF"
                     # Replace / with - in branch name
-                    echo docker_image_tag=${GITHUB_BASE_REF/\//-} >> $GITHUB_OUTPUT
+                    docker_image_tag=${GITHUB_BASE_REF/\//-}
                 fi
             fi
         else 
             if [[ "$GITHUB_REF_NAME" =~ release/(.*) ]]; then
-                echo docker_organization="usdotfhwastolcandidate" >> $GITHUB_OUTPUT
-                echo docker_image_tag="${BASH_REMATCH[1]}" >> $GITHUB_OUTPUT
-            elif [ "$GITHUB_REF_NAME" == "carma-master" ] || [ "$GITHUB_REF_NAME" == "master" ] || [[ "$GIT_TAG" != "" ]]; then
-                echo docker_organization="usdotfhwastol" >> $GITHUB_OUTPUT
-                echo docker_image_tag="$GIT_TAG" >> $GITHUB_OUTPUT
+                echo "docker_organization=usdotfhwastolcandidate" >> $GITHUB_OUTPUT
+                docker_image_tag="${BASH_REMATCH[1]}"
+            elif [ "$GITHUB_REF_NAME" == "carma-master" ] || [ "$GITHUB_REF_NAME" == "master" ] || [[ -n "$GIT_TAG" ]]; then
+                echo "docker_organization=usdotfhwastol" >> $GITHUB_OUTPUT
+                docker_image_tag="$GIT_TAG"
             else
-                echo docker_organization="usdotfhwastoldev" >> $GITHUB_OUTPUT
+                echo "docker_organization=usdotfhwastoldev" >> $GITHUB_OUTPUT
                 if [ "$GITHUB_REF_NAME" == "carma-develop" ] || [ "$GITHUB_REF_NAME" == "develop" ]; then
-                    echo docker_image_tag="develop" >> $GITHUB_OUTPUT
+                    docker_image_tag="develop"
                 else
                     # Replace / with - in branch name
-                    echo docker_image_tag=${GITHUB_REF_NAME/\//-} >> $GITHUB_OUTPUT
+                    docker_image_tag=${GITHUB_REF_NAME/\//-}
                 fi
             fi
         fi
+        
+        # Append the docker_tag_suffix if it's not empty
+        if [ -n "$DOCKER_TAG_SUFFIX" ]; then
+            docker_image_tag="${docker_image_tag}-${DOCKER_TAG_SUFFIX}"
+        fi
+        
```

Comment:
As the Docker image tag is now dynamically generated with an optional suffix, can we log the full Docker image tag, including the suffix for easier debugging during CI/CD runs like echo "Full Docker image tag: ${docker_image_tag}" some thing like that just a suggestion. everything looks good

Reply:
Yeah there is a final step to echo the changes and output:
`echo "Docker organization $DOCKER_ORGANIZATION and docker image tag $DOCKER_IMAGE"` 
which will print both.